usi/src/main/java/daq/usi/ISwitchController.java

```diff
@@ -0,0 +1,19 @@
+package daq.usi;
+
+import grpc.Interface;
+import grpc.Power;
+import grpc.SwitchActionResponse;
+
+public interface ISwitchController {
```

Comment:
What is ISwitchController?  Would like a more self-explanatory name and in general avoid Hungarian notation (e.g. I if it stands for Interface).  It might be that this is just the "SwitchController" while the other class is then the "BaseSwitchController"